src/vitest.ts

```diff
@@ -0,0 +1,11 @@
+import { DefaultReporter } from 'vitest/reporters';
+
+/**
+ * This reporter provides a real-time summary and reports errors in detail, but
+ * skips the individual file output.
```

Comment:
```suggestion
 * This reporter provides a real-time progress summary during test execution and reports failures in detail
 * but avoids printing a list of successful tests.
```

Is this an accurate description of what this reporter does?

Reply:
Yep